src/app/static/src/app/components/adr/SelectRelease.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+    <div id="selectRelease" v-if="newDatasetId && releases.length">
+        <div class="pt-1">
+            <input
+                type="radio"
+                id="useData"
+                value="useData"
```

Comment:
Would 'useLatest' be a little more meaningful than 'useData' - both choices use data...